README.md

```diff
@@ -1,6 +1,6 @@
 # PenguBot [![Build Status](https://travis-ci.org/AdityaTD/PenguBot.svg?branch=master)](https://travis-ci.org/AdityaTD/PenguBot) [![Discord](https://discordapp.com/api/guilds/303195322514014210/embed.png)](https://discord.gg/u8WYw5r) [![Greenkeeper badge](https://badges.greenkeeper.io/AdityaTD/PenguBot.svg)](https://greenkeeper.io/) [![Servers](https://discordbots.org/api/widget/servers/303181184718995457.svg?noavatar=true)](https://discordbots.org/bot/303181184718995457/vote)
 
-PenguBot is a community developed Discord Bot with over 19,000 guilds.
+PenguBot is a community developed Discord Bot with over more than 23,000 guilds.
```

Comment:
how does "with over more than" make sense?

Reply:
R.I.P. My Inglishhhh!
Ma Self Fram Village Area ;-;
But u got what i meant to do with this PR so please merge it or do required changes sir!
Thanks Sir,
With Regards
Piyush